libraries/AP_HAL_ChibiOS/hwdef/HerePro/hwdef.dat

```diff
@@ -11,7 +11,6 @@ USB_STRING_MANUFACTURER "CubePilot"
 
 # setup build for a peripheral firmware
 env AP_PERIPH 1
-env AP_PERIPH_HEAVY 1
```

Comment:
we can remove HerePro completely
@bugobliterator will create a HereProAP hwdef.dat for conventional use of HerePro as a DroneCAN GPS

Reply:
HerePro removed